content/chainguard/chainguard-images/reference/node/getting-started-node.md

```diff
@@ -0,0 +1,176 @@
+
+In this guide, we'll set up a demo application and create a Dockerfile to build and execute the demo using the Node Chainguard Image as base.
+
+This tutorial requires Docker, Node, and Npm to be installed on your local machine.
+
+{{< details "What is distroless" >}}
+{{< blurb/distroless >}}
+{{< /details >}}
+
+{{< details "What is Wolfi" >}}
+{{< blurb/wolfi >}}
+{{< /details >}}
+
+{{< details "Chainguard Images" >}}
+{{< blurb/images >}}
+{{< /details >}}
+
+## Step 1: Setting up a Demo Application
+
+We'll start by creating a basic Node application to serve as a demo. This application uses a mock server to test GET and POST requests, and is based on the [Docker Node example](https://docs.docker.com/language/nodejs/build-images/).
+
+First, create a directory for your app. In this guide we'll use `wolfi-node`:
+
+```shell
+mkdir ~/wolfi-node && cd ~/wolfi-node
+```
+
+Run the following command to create a new `package.json` file.
+
+```shell
+npm init -y
+```
+
+Next, install the application dependencies. We'll need `ronin-server` and `ronin-mocks`, which are used together to create a "mock" server that saves JSON data in memory and returns it in subsequent GET requests to the same endpoint.
+
+```shell
+npm install ronin-server ronin-mocks
+```
+
+Using your code editor of choice, create a new file named `server.js` for your application. Here we'll use `nano`:
+
+```shell
+nano server.js
+```
+
+Copy the following code to your `server.js` file:
+
+```js
+const ronin = require('ronin-server')
+const mocks = require('ronin-mocks')
+
+const server = ronin.server()
+
+server.use('/', mocks.server(server.Router(), false, true))
+server.start()
+```
+
+Save the file when you're done. Then, run the server with:
+
+```shell
+node server.js
+```
+
+This command will start the application and wait for connections on port `8000`. The mocking server will save any JSON data submitted by a POST request
+
+From a new terminal window, run the following command, which will make a POST request to your application sending a JSON payload:
+
+```shell
+curl --request POST \
+  --url http://localhost:8000/test \
+  --header 'content-type: application/json' \
+  --data '{"msg": "testing" }'
+```
+
+When the connection is successful, you should get output like this on the terminal that is running the application:
+
+```shell
+2023-02-07T15:48:54:2450  INFO: POST /test
+```
+You can now test that the content was saved by running a GET request to the same endpoint:
+
+```shell
+curl http://localhost:8000/test
+```
+
+You'll get output similar to this:
+
+```shell
+{"code":"success","meta":{"total":1,"count":1},"payload":[{"msg":"testing","id":"f427f835-3e93-43ad-91c8-d150dffba0f9","createDate":"2023-02-07T14:48:54.256Z"}]}
+```
+
+The demo application is now ready. In the next step, you'll create a Dockerfile to run your app.
```

Comment:
I went ahead with the next section and got this message:

```
docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:8000 -> 0.0.0.0:0: listen tcp 0.0.0.0:8000: bind: address already in use.
ERRO[0000] error waiting for container: context canceled
```

So I think a sentence or two here at the end to remind the user to CTRL/CMD+C their running `node server.js` would be helpful.